docs/usage/configuration-options.md

```diff
@@ -190,6 +190,10 @@ So for example you could choose to automerge all (passing) `devDependencies` onl
 }
 ```
 
+<!-- prettier-ignore -->
+!!! note
+    Branch automerge ignores the normal schedule and only follows the [`automergeSchedule`](#automergeschedule).
```

Comment:
Are you sure we don't follow the `schedule` when it comes to _creating_ the branch to be automerged?

Reply:
Branch creation still depends on `schedule`

Reply:
I think this comment is potentially confusing